modules/base/src/main/scala/io/janstenpickle/trace4cats/base/context/Init.scala

```diff
@@ -0,0 +1,36 @@
+package io.janstenpickle.trace4cats.base.context
+import cats.{~>, Monad}
+
+trait Init[Low[_], F[_], R, R0] extends Provide[Low, F, R] {
```

Comment:
As well, I don't see any need to combine a resource provider with the `Provide` typeclass.